apps/storefront/src/components/Titles.jsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/prop-types */
+import React from 'react'
+import { Typography } from '@equinor/eds-core-react'
+import styled, { css } from 'styled-components'
+import { slugify } from '../utils/'
+
+const tocAdjustment = css`
+  display: block;
+  content: ' ';
+  margin-top: -80px;
+  height: 80px;
+  visibility: hidden;
+`
+
+const StyledH1 = styled(Typography)`
```

Comment:
Should these changes be added to the `<Typography>` component? 🤔

Reply:
Maybe… 💭